spec/spec_helper.rb

```diff
@@ -41,6 +41,7 @@
 RSpec.configure do |config|
   config.before(:each) do
     setup_test_database
+    add_row_to_test_database
```

Comment:
Well done getting this to work.  If you imagine a new developer trying to get to grips with this codebase though, would it be clearer to call `add_row_to_test_database` in the spec file?